lib/core/resolvers/GitResolver.js

```diff
@@ -26,6 +26,9 @@ function GitResolver(decEndpoint, config, logger) {
     // anyway
     mkdirp.sync(config.storage.empty);
     process.env.GIT_TEMPLATE_DIR = config.storage.empty;
+    if (!config.strictSsl) {
+        process.env.GIT_SSL_NO_VERIFY = 'true';
+    }
```

Comment:
I think we need "else" part as well to support api interface.. Currently in API interface it's possible to run multiple bower commands in one process.